views/layouts/main.handlebars

```diff
@@ -88,7 +100,7 @@
     </div>
   </div>
   {{> footer}}
-
+<script src="public/js/myScripts.js"></script>
```

Comment:
Could you name it more convenient? Like `spinner.js`, so other contributors will know what is the function of the file